.github/workflows/gamescope-session.yml

```diff
@@ -1,25 +1,17 @@
 ---
-name: Upload gamescope-session source RPM
+name: Test gamescope-session
 on:
   push:
     branches:
-      - main
+      - !main
     paths:
       - 'gamescope-session/**'
 jobs:
-  build:
-    uses: playtron-os/rpm-specs-gaming/.github/workflows/all.yml@main
-  upload:
-    name: Upload gamescope-session source RPM to Fedora Copr
+  test:
+    name: Test gamescope-session RPM build
     runs-on: ubuntu-20.04
```

Comment:
ubuntu 22 too new? no issue from my view, just curious (since we're going with the latest fedora) I guess an argument could be made to build on older OS's as well to make sure all is working - that said maybe we should at some point define a set of supported OS's/versions

Reply:
No particular reason for using Ubuntu 20.04. I'll update to 22.04.